src/index.css

```diff
@@ -1,3 +1,277 @@
 * {
+  margin: 0;
+  padding: 0;
+  background-color: #f2f3f7;
+}
+
+.app {
+  box-sizing: border-box;
+  margin: 9.313rem 6.25rem 11.75rem 6.25rem;
+  max-width: 87.5rem;
+}
+
+.app div,
+.app,
+.book-div,
+.form,
+.form h2 {
+  background-color: #fafafa;
+}
+
+.flex {
+  display: flex;
+  justify-content: space-between;
+}
+
+.links ul {
+  list-style: none;
+}
+
+.books {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.links a {
+  text-decoration: none;
+  color: #969696;
+  background-color: white;
+  padding-left: 2.563rem;
+  font-size: 0.937rem;
+}
+
+.books > a.active,
+.categories > a.active {
+  color: #585858;
+}
+
+.nav div,
+.nav {
+  height: 5.875rem;
+  background-color: white;
+}
+
+.logo,
+.icon,
+.links {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo p,
+.icon p {
+  background-color: white;
+}
+
+.nav {
+  padding-inline: 6.25rem;
+  margin-bottom: 2.313rem;
+}
+
+.logo p {
+  font-size: 2.249rem;
+  font-weight: 900;
+  color: #0290ff;
+}
+
+.hr {
+  padding-inline: 6.25rem;
+  max-width: 75rem;
+  margin-bottom: 1.813rem;
+}
+
+.hr hr {
+  border: 1px solid #969696;
+}
+
+.form {
+  padding-inline: 6.25rem;
+  max-width: 75rem;
+}
+
+.form h2 {
+  font-size: 1.5rem;
+  color: #c9c9c9;
+  font-weight: 900;
+  margin-bottom: 1.188rem;
+}
+
+.form input {
+  margin-right: 2.125rem;
+  background-color: white;
+  border: 1px solid #969696;
+  margin-bottom: 16.563rem;
+  padding-left: 1.063rem;
+  font-size: 1.188rem;
+  color: #969696;
+}
+
+input.title {
+  width: 35.688rem;
+  height: 2.813rem;
+}
+
+input.author {
+  width: 17.813rem;
+  height: 2.813rem;
+}
+
+.add-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  color: white;
+  background-color: #0290ff;
+  border: none;
+  font-size: 1.125rem;
+}
+
+.progress-btn {
+  height: 2.813rem;
+  color: white;
+  background-color: #0290ff;
+  border: none;
+  font-size: 1.125rem;
+  padding-left: 1.375rem;
+  padding-right: 1.188rem;
+}
+
+.add-btn:hover,
+.progress-btn:hover {
+  color: #0290ff;
+  background-color: white;
+  cursor: pointer;
+  border: 1px solid #0290ff;
+}
+
+.add-btn:active,
+.progress-btn:active {
+  color: white;
+  background-color: #0290ff;
+  cursor: pointer;
+}
+
+.book-list {
+  padding-inline: 6.25rem;
+  max-width: 75rem;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 3fr 2fr 2fr;
+  margin-bottom: 0.938rem;
+}
+
+.list {
+  height: 10.625rem;
+  margin-bottom: 0.938rem;
+  padding-left: 1.688rem;
+  padding-bottom: 1.75rem;
+  padding-top: 2rem;
+}
+
+div.list,
+div.btn-div,
+div.percentage,
+div.card {
+  background-color: white;
+}
+
+p.action {
+  font-size: 1.125rem;
+  background-color: white;
+  color: #c9c9c9;
+  margin-bottom: 0.25rem;
+}
+
+p.book-title {
+  background-color: white;
+  color: black;
+  font-weight: 900;
+  margin-bottom: 0.25rem;
+  font-size: 1.813rem;
+}
+
+p.book-author {
+  background-color: white;
+  color: #0290ff;
```

Comment:
- [ ] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.